app.rb

```diff
@@ -12,19 +12,6 @@ def initialize
     @rentals = []
   end
 
-  def home_page
-    puts 'Welcome to Library App!'
-    until list_of_options
-      input = gets.chomp.to_i
-      if input == 7
-        puts 'Thank you for using Library App!'
-        exit
-      end
-
-      option input
-    end
-  end
-
   def list_all_persons
     puts 'Person List is empty. Please add a person first.' if @persons.empty?
     @persons.each do |person|
```

Comment:
- [ ]  Great job to optimize your app in app.rb but there are things that need to be fixed.  When I run the app, and create person or book, nothing is being created in the console.  please fix the issues so that the app is working as it expected to work. Match this  [videos](https://www.youtube.com/watch?v=vkkgrhD6aXQ&ab_channel=EmiliaAndrzejewska). to fulfill the required requirement.

![image](https://github.com/yosaddis/School-library-Ruby/assets/43877689/ed6b763c-3136-4578-a36b-4a05b324a3b1)